src/rust/ide/lib/view/src/graph_editor/component/documentation_view/style.css

```diff
@@ -0,0 +1,423 @@
+.docVis {
```

Comment:
there is a LOT of hardcoded values here. Can we make this as variables using less scss or something ?

Reply:
Yes, it could be, actually this is generated from .sass file i've created way back in September, but it would need a compilation process ...

Reply:
And i feel like this file will undergo MAAANY changes in my next task, so can we leave it for now as it is please?